### PR TITLE
added the milestone requirements

### LINKED ISSUE #24: Updating the readme

### PR DESCRIPTION
**Pull request by atabak-alishiri**

### CODE REVIEW

**Review by SoloSynth1 — COMMENTED** [review]
I think we will have to include the actual function names as a list.

The requirement states:
> - a bulleted list of the functions (and datasets if applicable) that will be included in the package (this should be a 1-2 sentence description for each function/dataset)

**Comment by atabak-alishiri** [comment]
> I think we will have to include the actual function names as a list.
> 
> The requirement states:
> 
> > * a bulleted list of the functions (and datasets if applicable) that will be included in the package (this should be a 1-2 sentence description for each function/dataset)

I will be able to edit that as soon as all the necessary functions are available.

**Comment by SoloSynth1** [comment]
No problem. I can get it ready before 4pm today.

**Comment by SoloSynth1** [comment]
@atabak-alishiri the functions should be ready now, thanks

**Comment by atabak-alishiri** [comment]
> @atabak-alishiri the functions should be ready now, thanks

@SoloSynth1 Now it has the function and class names.

File: README.md [hunk 1]
```diff
@@ -16,9 +16,23 @@ In alphabetical order:
 $ pip install visumorph
 ```
 
-## Usage
+## Features
 
-- TODO
+VisuMorph includes a variety of functions for image manipulation:
+
+- **Image Class(`Image`)**: A core class that represents images, facilitating the application of transformations and adjustments.
```
Comment (SoloSynth1) [inline README.md]: I think we better not include this as this is a class (not a function) and it is not "user-facing".

```suggestion
```

**Review by SoloSynth1 — APPROVED** [review]
@atabak-alishiri Please see comment. Otherwise, I think everything LGTM

**Pull request merged**